tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -532,3 +552,1116 @@ Feature: an user gets the resources shared to them
        }
      }
     """
+
+
+  Scenario: user lists the file shared with them from project space
+    Given using spaces DAV path
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "new-space" with content "some content" to "testfile.txt"
+    And user "Alice" has sent the following share invitation:
+      | resource        | testfile.txt |
+      | space           | new-space    |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | Viewer       |
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+    """
+    {
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": [
+              "@UI.Hidden",
+              "@client.synchronize",
+              "eTag",
+              "file",
+              "id",
+              "lastModifiedDateTime",
+              "name",
+              "parentReference",
+              "remoteItem"
+            ],
+            "properties": {
+              "@UI.Hidden":{
+                "type": "boolean",
+                "enum": [false]
+              },
+              "@client.synchronize":{
+                "type": "boolean",
+                "enum": [true]
+              },
+              "eTag": {
+                "type": "string",
+                "pattern": "%eTag%"
+              },
+              "file": {
+                "type": "object",
+                "required": ["mimeType"],
+                "properties": {
+                  "mimeType": {
+                    "type": "string",
+                    "enum": ["text/plain"]
+                  }
+                }
+              },
+              "id": {
+                "type": "string",
+                "pattern": "^%share_id_pattern%$"
+              },
+              "name": {
+                "type": "string",
+                "enum": [
+                  "testfile.txt"
+                ]
+              },
+              "parentReference": {
+                "type": "object",
+                "required": [
+                  "driveId",
+                  "driveType",
+                  "id"
+                ],
```

Comment:
for `parentReference` do we get like this in response https://github.com/owncloud/ocis/blob/4d5490a60135df8beb8a64626192e6555858f74e/tests/acceptance/features/apiSharingNg/sharedByMe.feature#L37-L45. may be we could also extend to it if we get them in the reponse. For other response also.